File: wk6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

row = 0
for i in range(height):
        print(" " * (height - 1 - row) , end='')
        print("#" * (row + 1) , end=' ')
        print(end=' ')
        print("#" * (row + 1))
        row += 1

File: wk1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h, r, c, d;
    do
    {
        h = get_int("Height: ");
    }
    while ( h < 1 || h > 8 );

    // for columns
    for ( c = 0; c < h; c++)
    {
        // for rows (dots)

        for ( d = h - 1 ; d > c ; d--)
          {
            printf(" ");
          }
        // for rows (#)
        for ( r = 0; r <= c; r++)
        {
          printf("#");
        }
        printf("\n");
    }
}
